Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```

Comment:
Accidentally left this in. Bundle install doesn't work with https for me for some reason.